src/Poseidon/Package.hs

```diff
@@ -280,3 +330,62 @@ compFunc1 (EigenstratSnpEntry c1 p1 _ _ _ _, _) (EigenstratSnpEntry c2 p2 _ _ _
 compFunc2 :: (EigenstratSnpEntry, GenoLine) -> [(EigenstratSnpEntry, GenoLine)] -> Ordering
 compFunc2 (EigenstratSnpEntry c1 p1 _ _ _ _, _) ((EigenstratSnpEntry c2 p2 _ _ _ _, _):_) = compare (c1, p1) (c2, p2)
 compFunc2 _                                     []                                        = error "compFunc2 - should never happen"
+
+-- Janno file loading
+
+decodingOptions :: Csv.DecodeOptions
+decodingOptions = Csv.defaultDecodeOptions { 
+    Csv.decDelimiter = fromIntegral (ord '\t')
```

Comment:
This is bizarre... `fromIntegral . ord` must be basically `id`. I'll have a look what type `Csv.decDelimiter` needs to be.

Reply:
OK, found it. The docs even recommend this, because the type of `Csv.decDelimiter` is a `Word8`. Fine.